bridgebot/enums.py

```diff
@@ -124,12 +132,27 @@ def is_player_in_team(self, player):
             raise InvalidPlayerException("Invalid Player")
         return player in self.to_set_of_players()
 
+
 class Vulnerabilities(Enum):
     NONE = 'NONE'
     NS = 'NS'
     EW = 'EW'
     BOTH = 'BOTH'
 
+    def is_declarer_vulnerable(self, declarer):
```

Comment:
The scoring functions are cleaner when you can pass them a `bool` instead of a `Vulnerabilities`